packages/server/package.json

```
@@ -119,8 +124,9 @@
     "signal-exit": "3.0.2",
     "sinon": "5.1.1",
     "squirrelly": "7.7.0",
-    "strip-ansi": "3.0.1",
+    "strip-ansi": "6.0.0",
     "syntax-error": "1.4.0",
+    "systeminformation": "^4.16.0",
```

Should be pinned dep to avoid breaking updates.

```suggestion
    "systeminformation": "4.16.0",
```